packages/@vuepress/core/lib/client/root-mixins/updateMeta.js

```diff
@@ -64,3 +76,35 @@ function updateMetaTags (meta, current) {
     })
   }
 }
+
+/**
+ * Try to identify a meta tag by name, property or itemprop
+ *
+ * Return a complete string if none provided
+ * @param {Object} tag from frontmatter or siteMetaTags
+ * @returns {String}
+ */
+function metaIdentifier (tag) {
+  for (const item of ['name', 'property', 'itemprop']) {
+    if (tag.hasOwnProperty(item)) return tag[item] + item
+  }
+  return JSON.stringify(tag)
+}
+
+/**
+ * Render meta tags
+ *
+ * @param {Array} meta
+ * @returns {Array<string>}
+ */
+
+function renderPageMeta (meta) {
+  if (!meta) return ''
+  return meta.map(m => {
+    let res = `<meta`
+    Object.keys(m).forEach(key => {
+      res += ` ${key}="${m[key]}"`
```

Comment:
Shouldn't this be escaped? For example, this `config.js`:
```js
module.exports = {
  description: 'Image cache & resize service',
}
```

renders as (within version 1.4.1):
```html
    <meta name="description" content="Image cache & resize service">
```

which [is probably not correct](https://stackoverflow.com/a/20585780/10952119). fwiw, version 1.4.0 didn't had this problem.

Suggestion:
```suggestion
      res += ` ${key}="${escape(m[key])}"`
```
and this import at the top of the file:
```js
const escape = require('escape-html')
```